tests/integration_tests/bugs/test_gh626.py

```diff
@@ -0,0 +1,43 @@
+"""Integration test for gh-626.
+
+Ensure if wakeonlan is specified in the network config that it is rendered
+in the /etc/network/interfaces or netplan config.
+"""
+
+import pytest
+import yaml
+
+from tests.integration_tests.clouds import ImageSpecification
+from tests.integration_tests.instances import IntegrationInstance
+
+
+NETWORK_CONFIG = """\
+version: 2
+ethernets:
+  eth0:
+    dhcp4: true
+    wakeonlan: true
+"""
+
+EXPECTED_ENI_END = """\
+iface eth0 inet dhcp
+    ethernet-wol g"""
+
+
+@pytest.mark.sru_2020_11
+@pytest.mark.ubuntu
+@pytest.mark.lxd_container
+@pytest.mark.lxd_vm
+@pytest.mark.lxd_config_dict({
+    'user.network-config': NETWORK_CONFIG
+})
+def test_wakeonlan(client: IntegrationInstance):
+    if ImageSpecification.from_os_image().release == 'xenial':
```

Comment:
Checking this way isn't ideal, but I see it as being short term until we come up with a better way of checking

Reply:
I'm not even sure if this is desirable, but this would DTRT even if we haven't detected a specified image as `xenial`:

```suggestion
    if client.execute("lsb_release --short --codename") == "xenial":
```

(Presumably "a better way" would be separate tests with appropriate release tagging?)